src/contributors/yusufalp.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Yusuf Alp", 
+  "desc": "Front end web developer who enjoys puzzles and coding.",
+  "github" : "https://github.com/yusufalp",
```

```suggestion
  "github": "https://github.com/yusufalp",
```
Remove unwanted `white-space`.